site/docs/guidelines/inclusive_design.mdx

```diff
@@ -0,0 +1,70 @@
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Link from "../../src/components/Link";
+
+# Inclusivity
+<StatusLabel type="alert">Draft</StatusLabel>
+
+<LargeParagraph>
+  The city of Helsinki aims to do ambitious inclusivity work to promote equality for all of its citizens. The goal is to advance equality in the city of Helsinki services and activities. This page guides how to design and develop inclusive services with Helsinki Design System.
+</LargeParagraph>
+
+The content of this page is based on "Towards an Equal Helsinki" document published and ratified by the City of Helsinki council. Helsinki's inclusivity work is based on equality laws, expertise, cooperation, science-based information as well as city values and strategy. Please refer to the original document if you have questions about the inclusivity in Helsinki:
+- <Link href="hhttps://hel.fi/static/liitteet-2019/Kaupunginkanslia/Tasa-arvo/plan-for-promoting-gender-equality-helsinki.pdf" external>Towards an Equal Helsinki document (in English)</Link>
+- <Link href="https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Tasa-arvo/palveluiden-tasa-arvo-suunnitelma-helsinki-2019-2021.pdf" external>Kohti tasa-arvoista Helsinkiä document (in Finnish)</Link>
+
+## What is inclusive design?
+Inclusive design means taking into account the needs of the users who might experience exclusion in their daily lives. This exclusion may be due to being part of a specific group or minority. The goal is to intentionally include these groups to reduce the risk of unintentionally excluding them.
+
+In Helsinki, we aim to apply inclusive design in all parts of the service development. Not only this means designing, developing, testing, and releasing the service, but also everything related to it such as meetings, workshops, and daily interactions with other people. 
+
+As stated in the Helsinki equality plan, "_The most important goal of all gender equality work is to make Helsinki develop into a city where people representing all genders, gender identities, and gender expressions can be seen and heard and are allowed to participate and act as equal citizens without direct or indirect discrimination_". 
+
+## Setting up your project for success
+When starting a new project, there are two (2) main elements that it comes to ensure inclusivity. 
+
+### 1. Multidisciplinary and diverse team
+The structure of your project team shapes the decision it makes. Always aim to have a multidisciplinary team and make sure your team members can work together. It is ideal if all team members can be part of the project right from the research and design phase to the testing and release.
+
+While it is not always possible to ensure your team is diverse enough, it is important to acknowledge the diversity of your team. In small non-diverse teams, it is important to make sure the designs and implementations are tested with a more diverse group of test users.
```

Comment:
Maybe a bit hard to understand. Could it be "It is not always possible to ensure that your team is diverse enough but it is important to acknowledge the diversity of your team. And especially in small non diverse teams, it is important....